pandas/core/reshape/merge.py

```diff
@@ -159,13 +159,13 @@ def merge_ordered(left, right, on=None,
         left DataFrame
     fill_method : {'ffill', None}, default None
         Interpolation method for data
-    suffixes : Sequence, default is ("_x", "_y")
+    suffixes : Sequence or False/None, default is ("_x", "_y")
```

Comment:
we don't want to accept False, rather this is an optional argument

Reply:
okay... will remove False option... thanks @jreback